### PR TITLE
[Cortx] EOS-14222UI text review changes for preboarding and onboarding page

### PR DESCRIPTION
**Pull request by namrata-khake-ui**
# UI

text review changes

## Problem Statement
<pre>
  <code>
    Story Ref (if any):EOS-14222
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  No
  </code>
</pre>
## Problem Description
<pre>
  <code>
1. Welcome!!! changed to Welcome!
2. Changed "Click on Start to proceed with on onboarding steps." to "Click Start to proceed with onboarding steps."
3. username info tooltip : one space before this sentence "The username must not be 'root' or 'Root'."
4.password info tooltip: change from "Password must contain: Minimum 8 characters, One uppercase letter, One lowercase letter, One special character, One number"
   to "Password must contain: Minimum 8 characters, One uppercase letter, One lowercase letter, One special character, and One number"
5.DNS setting
    there should be little bit space above "DNS server" and "search domains" inputs
  </code>
</pre>
## Solution/Screenshots
image
image
image
image

<pre>
  <code>
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
  tested on locally
  </code>

### CODE REVIEW

**Review by JayshreeAmore — APPROVED** [review]

File: gui/src/common/regex-helpers.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export const accountNameTooltipMessage =
 
 export const usernameTooltipMessage =
   // tslint:disable-next-line
-  "The username must be of minimum 4 characters and maximum 56 characters. The username must be alphanumeric and can contain underscore (_) and dash (-).The username must not be 'root' or 'Root'.";
+  "The username must be of minimum 4 characters and maximum 56 characters.  The username must be alphanumeric and can contain underscore (_) and dash (-).The username must not be 'root' or 'Root'.";
```
Comment (giftedUI) [inline gui/src/common/regex-helpers.ts:28]: Why this change, can we revert this line

**Review by giftedUI — APPROVED** [review]

**Review by kupranay — APPROVED** [review]
